docs/content/tutorial/managing-your-own-io.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Tutorial, part seven: Managing your own I/O | Dagster Docs"
+description: Learn how to use I/O managers to save your data.
+---
+
+# Tutorial, part 7: Managing your own I/O
+
+When building the dictionary or word counts, the `most_frequent_words` asset currently uses a static list of stopwords to decide what words to ignore. You’ll notice that despite the length of the list, many unhelpful words and prepositions are still not filtered out. You’d like to ignore those words. To scale this properly, you’ll use a CSV of stopwords found on the internet, use it for `most_frequent_words`, and update it before the refresh to ensure that your list of stopwords is up-to-date.
```

Comment:
Feel free to push back on this, but my instinct would be to basically put the second paragraph first.  I.e. start by motivating what this section of the tutorial is about before getting into the nitty gritty.

Reply:
Astute of you. I was going back and forth on this during drafting, so I'm glad that the value of one over the other wasn't just in my head.